### ISSUE TITLE
issue debugging when more then 1 worker pool exists

### ISSUE THREAD

**Issue by stefanpenner** [josdejong/workerpool#59]
Example:

```js
// index.js
'use strict';

const workerpool = require('workerpool');
const pool = workerpool.pool();
const pool2 = workerpool.pool();

const add = (a, b) => a + b;

Promise.all([
  pool.exec(add, [1, 1]),
  pool2.exec(add, [1, 1])
])
  .then(function (result) {
    console.log('result', result); // outputs 7
  })
  .catch(function (err) {
    console.error(err);
  }).then(() => {
    pool.terminate();
    pool2.terminate();
  });
```

Running this with:

```sh
node ./index.js
```

Worker correctly.

Unfortunately, running this with combined with https://github.com/josdejong/workerpool/pull/58 for better error message output

```sh
node --inspect ./index.js
```

results in:

```sh
node --inspect foo.js
Debugger listening on ws://127.0.0.1:9229/194b112a-6b22-4abf-8ad9-4aaa4920475b
For help see https://nodejs.org/en/docs/inspector
Debugger listening on ws://127.0.0.1:43210/a7b46e61-8702-4334-9281-9467bf55c9e5
For help see https://nodejs.org/en/docs/inspector
Starting inspector on 127.0.0.1:43210 failed: address already in use
Error: Workerpool Worker terminated Unexpectedly
    exitCode: `12`
    signalCode: `null`
    workerpool.script: `/Users/spenner/src/josdejong/workerpool/lib/worker.js`
    spawnArgs: `/Users/spenner/src/stefanpenner/dotfiles/.config/node/versions/node-v8.9.4-darwin-x64/bin/node,--inspect=43210,/Users/spenner/src/josdejong/workerpool/lib/worker.js`
    spawnfile: `/Users/spenner/src/stefanpenner/dotfiles/.config/node/versions/node-v8.9.4-darwin-x64/bin/node`
    stdout: `null`
    stderr: `null`

    at ChildProcess.<anonymous> (/Users/spenner/src/josdejong/workerpool/lib/WorkerHandler.js:259:13)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
```

As both subprocess's get attempt to bind to the same inspector port `--inpect=43210`

**Comment by stefanpenner** [comment]
fix: https://github.com/josdejong/workerpool/pull/60

**Comment by josdejong** [comment]
This issue should have been closed already, the fix is available since v3.1.2.
